packages/react/src/components/accordion/accordion-types.ts

```diff
@@ -0,0 +1,31 @@
+import { ReactNode, RefObject, KeyboardEvent } from 'react';
+import { Type, Tag } from '../heading/heading';
+
+export interface AccordionProps {
+    /** Unique id per accordion */
+    id: string;
+    children: ReactNode;
+    /** Multipe or single panel open concurrently */
+    mode?: 'single' | 'multi';
+}
+
+export interface AccordionItemProps {
+    /** Title Label */
```

Comment:
Not sure this comment adds something

Reply:
this comment is going into storybook when displaying the props there.
I believe they are removed when compile the project to final use right?

Reply:
removed these comments that was used on story book